From 58381f26c288505766daac9e0f20f879fcd06631 Mon Sep 17 00:00:00 2001
From: Richard Levitte <levitte@openssl.org>
Date: Sat, 24 Nov 2018 00:59:33 +0100
Subject: [PATCH] Don't export the submodules 'boringssl', 'krb5' and
 'pyca-cryptography'

Reviewed-by: Matthias St. Pierre <Matthias.St.Pierre@ncp-e.com>
(Merged from https://github.com/openssl/openssl/pull/7696)

(cherry picked from commit 76bc401cc63219a462224884cb4af787e17725ed)
---
 .gitattributes | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/.gitattributes b/.gitattributes
index 0902ea422549..96e40b6cd45b 100644
--- a/.gitattributes
+++ b/.gitattributes
@@ -7,3 +7,6 @@ fuzz/corpora/**                         export-ignore
 Configurations/*.norelease.conf         export-ignore
 .*                                      export-ignore
 util/mktar.sh                           export-ignore
+boringssl                               export-ignore
+krb5                                    export-ignore
+pyca-cryptography                       export-ignore
